vault-init-eso.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+enable_eso() {
+    local eso_service_account_namespace
+    local eso_service_account_name
+    local eso_pod_name
+
+    export SA_JWT_TOKEN=$(kubectl -n vault get secret auth-sa-token -o jsonpath="{.data.token}" | base64 --decode; echo)
+    export SA_CA_CRT=$(kubectl -n external-secrets get secret auth-sa-token -o jsonpath="{.data['ca\.crt']}" | base64 --decode; echo)
+
+    kubectl exec vault-0 -n vault -- vault auth enable kubernetes
+
+    kubectl exec vault-0 -n vault -- vault write auth/kubernetes/config \
```

Comment:
At the moment Vault doesn't creating secret linked to service for Kubernetes auth automatically. It can be done via values:
```
vault:
  server:
    serviceAccount:
      createSecret: true
```
But general recommended approach to use short-lived token, that mounted directly into vault pod (https://developer.hashicorp.com/vault/docs/auth/kubernetes#how-to-work-with-short-lived-kubernetes-tokens)

So we can use that code without parsing secret data from kubectl:
```
vault write auth/kubernetes/config \
    kubernetes_host=https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT
```
This env should be accessible from vault-0 pod, so please make necessary changes.